src/neptune/new/types/type_casting.py

```diff
@@ -111,12 +121,13 @@ def cast_value_for_extend(values: Union[Namespace, Series, Collection[Any]]) ->
     elif is_float_like(sample_val):
         return FloatSeries(values=values)
     elif is_string_like(sample_val):
-        warn_once(
-            message="The object you're logging will be implicitly cast to a string."
-            " We'll end support of this behavior in `neptune-client==1.0.0`."
-            " To log the object as a string, use `str(object)` instead.",
-            stack_level=4,
-        )
+        if not from_stringify_value:
+            warn_once(
+                message="The object you're logging will be implicitly cast to a string."
+                " We'll end support of this behavior in `neptune-client==1.0.0`."
+                " To log the object as a string, use `str(object)` or"
+                " `stringify_unsupported(collection)` for collections and dictionaries instead."
```

Comment:
Remove "instead". Add link to https://docs.neptune.ai/setup/neptune-client_1-0_release_changes